src/components/DynamicWidgets.js

```diff
@@ -0,0 +1,89 @@
+import { createWidgetMixin } from '../mixins/widget';
+import { EXPERIMENTAL_connectDynamicWidgets } from 'instantsearch.js/es/connectors';
+import { createSuitMixin } from '../mixins/suit';
+
+function getVueAttribute(vnode) {
+  if (vnode.componentOptions && vnode.componentOptions.propsData) {
+    if (vnode.componentOptions.propsData.attribute) {
+      return vnode.componentOptions.propsData.attribute;
+    }
+    if (
+      vnode.componentOptions &&
+      Array.isArray(vnode.componentOptions.propsData.attributes)
+    ) {
+      return vnode.componentOptions.propsData.attributes[0];
+    }
```

Comment:
```suggestion
    if (Array.isArray(vnode.componentOptions.propsData.attributes)) {
      return vnode.componentOptions.propsData.attributes[0];
    }
```

The first condition doesn't seem necessary